pulumi/grapl/__main__.py

```diff
@@ -59,6 +59,7 @@ def _container_images(artifacts: ArtifactGetter) -> Mapping[str, DockerImageId]:
         "engagement-creator": builder.build_with_tag("engagement-creator"),
         "graph-merger": builder.build_with_tag("graph-merger"),
         "graphql-endpoint": builder.build_with_tag("graphql-endpoint"),
+        "hax-docker-plugin-runtime": DockerImageId("debian:bullseye-slim"),
```

Comment:
"sure, why not"

Reply:
maybe alpine is better, we want a pretty minimal image

Reply:
I think bullseye-slim is fine for now. Alpine can have issues with C dependencies, which can be problematic with certain python wheels

Reply:
Agree that debian-slim is fine